api.md

```diff
@@ -658,6 +680,8 @@ A 400 http status code and the errors in the response body.
 }
 ```
 
+An empty body or a JSON object with no `message` is returned on success.
```

Comment:
It would be nice to document which of the two it is based on the input

Reply:
Per the TODO in the API code, I want to change it later to return an object. For now I want to make people tolerant of either way.